app/assets/stylesheets/custom.css.scss

```diff
@@ -152,3 +152,36 @@ p {
     color: #333;
   }
 }
+
+/* forms */
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
+  width: 100%;
+  margin-bottom: 15px;
+  @include box_sizing;
+}
+
+input {
+  height: auto !important;
+}
+
+label.required:after {
+  color: $purple;
+  content:" *";
+}
+
+#error_explanation {
+  color: red;
+  border: 3px solid rgba(239, 121, 132, 0.9);
+  background: rgba(245, 168, 179, 0.54);
+  .alert.alert-error{
```

Comment:
Rule declaration should be preceded by an empty line<br>Opening curly brace `{` should be preceded by one space